src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActionsSource.cs

```diff
@@ -610,6 +614,8 @@ private void OnSuggestedActionsChanged(Workspace currentWorkspace, DocumentId? c
                 if (document == null)
                     return null;
 
+                var options = CodeActionOptionsFactory.GetCodeActionOptions(document.Project, isBlocking: true);
```

Comment:
isn't this *non* blocking?  we're in an async entrypoint?

Reply:
Good catch. Should be `false`